src/util/duration.h

```diff
@@ -136,19 +157,19 @@ class Duration {
     }
 
     QString formatNanosWithUnit() const {
-        return QString("%1ns").arg(toNanos());
+        return QString("%1ns").arg(toIntegerNanos());
```

Comment:
This is not a question of a personal preference of one of us. 50 s is correct 50s is wrong. See http://www.nist.gov/pml/wmd/metric/writing-metric.cfm "Spacing" 

After this PR the formating of the log file changes. Even if you do not touch these particular lines this is a regression caused by this PR. It is no problem for me to use the 50s format if we have a valid reason for it. But this should be the expectational case like formatNanosWithUnitNoSpace()

The controllerDebug line now always wraps, so there is no issue of a single space.